controllers/admin/AdminImportController.php

```diff
@@ -339,23 +547,52 @@ public function __construct()
-                    'date_add' => array('label' => $this->trans('Registration date (yyyy-mm-dd)', array(), 'Admin.Advparameters.Feature')),
-                    'group' => array('label' => $this->trans('Groups (x,y,z...)', array(), 'Admin.Advparameters.Feature')),
-                    'id_default_group' => array('label' => $this->trans('Default group ID', array(), 'Admin.Advparameters.Feature')),
+                    'id_gender' => array(
+                        'label' => $this->trans(
+                            'Titles ID (Mr = 1, Ms = 2, else 0)',
+                            array(),
+                            'Admin.Advparameters.Feature'
+                        ),
+                    ),
+                    'email' => array('label' => $this->trans('Email', array(), 'Admin.Global') . '*'),
+                    'passwd' => array('label' => $this->trans('Password', array(), 'Admin.Global') . '*'),
+                    'birthday' => array(
+                        'label' => $this->trans('Birth date (yyyy-mm-dd)', array(), 'Admin.Advparameters.Feature'),
+                    ),
+                    'lastname' => array('label' => $this->trans('Last name', array(), 'Admin.Global') . '*'),
+                    'firstname' => array('label' => $this->trans('First name', array(), 'Admin.Global') . '*'),
+                    'newsletter' => array(
+                        'label' => $this->trans('Newsletter (0/1)', array(), 'Admin.Advparameters.Feature'),
+                    ),
+                    'optin' => array(
+                        'label' => $this->trans('Partner offers (0/1)', array(), 'Admin.Advparameters.Feature'),
+                    ),
+                    'date_add' => array(
+                        'label' => $this->trans(
+                            'Registration date (yyyy-mm-dd)',
+                            array(),
+                            'Admin.Advparameters.Feature'
+                        ),
+                    ),
+                    'group' => array(
+                        'label' => $this->trans('Groups (x,y,z...)', array(), 'Admin.Advparameters.Feature'),
+                    ),
+                    'id_default_group' => array(
+                        'label' => $this->trans('Default group ID', array(), 'Admin.Advparameters.Feature'),
+                    ),
                     'id_shop' => array(
                         'label' => $this->trans('ID / Name of shop', array(), 'Admin.Advparameters.Feature'),
-                        'help' => $this->trans('Ignore this field if you don\'t use the Multistore tool. If you leave this field empty, the default shop will be used.', array(), 'Admin.Advparameters.Help'),
+                        'help' => $this->trans(
+                            'Ignore this field if you don\'t use the Multistore tool. If you leave this field empty, '
+                                . 'the default shop will be used.',
```

Comment:
Then maybe we are debating about the wrong subject then.
Ask me to break the standards and I'll do it.